lib/game_team.rb

```diff
@@ -33,7 +33,7 @@ def self.create(file_path)
                                     row[:settled_in],
                                     row[:head_coach],
                                     row[:goals].to_i,
-                                    row[:shots],
+                                    row[:shots].to_i,
```

Comment:
we might need to adjust game_team test for this change

Reply:
It looks like we aren't failing any of those tests.